docs/introduction/apphooks.rst

```diff
@@ -33,7 +33,9 @@ This is the most basic example of an apphook for a django CMS application:
 
     apphook_pool.register(PollsApphook)  # register the application
 
-Restart the runserver.
+**Restart the runserver**. This is necessary because we have created a new file containing Python
+code that won't be loaded until the server restarts. You only have to do this the first time the
+new file has been created.
```

Comment:
Actually, now that we have apphooks_reload functionality baked into the CMS, this may not be necessary anymore. Perhaps we should weaken this, just in case the user is on an exotic platform that still requires the reload?